index.js

```diff
@@ -1,4 +1,94 @@
-//  in this file create an express application - use the middle-ware built into express
-//  to serve up static files from the public directory (index.html and client.js - you 
-//  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
\ No newline at end of file
+'use strict';
+//  in this file create an express application
+//  use the middle-ware built into express
+//  to serve up static files from public directory (index.html & client.js)
+//  you can also serve up css files from the public directory  if you want)
+//  you need to support '/trucks' endpoint
+// and dynamic route for '/trucks/:name'
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var trucks = require('./trucks');
+var foodTypes = require('./trucks');
+var openTrucks = require('./trucks');
+var typeTrucks = require('./trucks');
+/* eslint-enable no-unused-vars */
+/* eslint-enable no-undef */
+
+
+var app = express();
+var serveStatic = express.static('public');
+
+var openTruck = trucks.filterTrucksByDay('Saturday');
+var pizzaTruckList = trucks.filterTrucksByFoodType('pizza');
+
+app.use(serveStatic);
+
+/* eslint-disable no-console */
+
+app.use(function (request, response, next) {
+  console.log('Request URL: ', request.originalURL);
+  next();
+});
+
+app.use(function (request, response, next) {
+  console.log('Time: ', Date.now());
+  next();
+});
+
+//   `/trucks` This route returns the list of all trucks in the module.
+// http://127.0.0.1:3000/trucks
+app.get('/trucks', function (request, response, next) {
+  var truckList = trucks.getTrucks();
+  response.send(truckList);
+  next();
+});
+
+// `/trucks/:name` This route returns a single truck object that matches
+// the name parameter passed in the route.
+// http://127.0.0.1:3000/trucks:Fez
+app.get('/trucks:name', function (request, response, next) {
```

Comment:
An example path this needs to match is `/trucks/314%20PIE`  Notice the slash between trucks and the parameter part of the path? Your path doesn't match, so it won't work for this example path.